src/features/workspace/chat/routes/ChatPage.tsx

```
@@ -1,157 +1,236 @@
   
   const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     if (event.target.files && event.target.files[0]) {
         setAttachedFile(event.target.files[0]);
     }
   };
+
+  // 채팅방 이름 표시 함수
+  const getChatRoomDisplayName = () => {
+    if (!state.currentChatRoom) return '채팅방';
+    
+    const room = state.currentChatRoom;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)